extensions/obviousAlexC/SensingPlus.js

```diff
@@ -178,6 +200,9 @@
 
   /** @param {TouchEvent} event */
   function handleTouchMove(event) {
+    if (hackyTouchFix) {

```

Comment:
is there a reason to not just *always* do this, and have you actually found a way to test this